client/python/cryoet_data_portal/pyproject.toml

```diff
@@ -1,10 +1,10 @@
 [build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
+requires = ["setuptools>=64"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "cryoet_data_portal"
-dynamic = ["version"]
+version = "3.0.3"
```

Comment:
Does release please update this?

Reply:
And the [src/cryoet_data_portal/__init__.py](https://github.com/chanzuckerberg/cryoet-data-portal/pull/972/files#diff-51bfa7b6ff3f0643c80e954b9686c748cf90b433b5e5822ab4b2cd0cf41d04a0) file.

Reply:
yes, it updates both